docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1371 @@
+{
```

Comment:
<p>I see that you reduced this line significantly to make the simulation possible. Maybe you could define the line_length during device creation, and have it 20 for simulation and 35 for other cases?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Done